.github/workflows/ci.yml

```diff
@@ -4,35 +4,103 @@ on:
   push:
     branches: [ master ]
   pull_request:
-    branches: [ main ]
+    branches: [ master ]
+
+env:
+  VTK_TAG: "v9.5.0.rc2"
+  VTK_MODULES: >-
+    -DVTK_GROUP_ENABLE_StandAlone=DONT_WANT
+    -DVTK_GROUP_ENABLE_Rendering=DONT_WANT
+    -DVTK_MODULE_ENABLE_VTK_ChartsCore=YES
+    -DVTK_MODULE_ENABLE_VTK_FiltersGeneral=YES
+    -DVTK_MODULE_ENABLE_VTK_FiltersHybrid=YES
+    -DVTK_MODULE_ENABLE_VTK_FiltersModeling=YES
+    -DVTK_MODULE_ENABLE_VTK_FiltersSources=YES
+    -DVTK_MODULE_ENABLE_VTK_GUISupportQt=YES
+    -DVTK_MODULE_ENABLE_VTK_ImagingColor=YES
+    -DVTK_MODULE_ENABLE_VTK_ImagingMorphological=YES
+    -DVTK_MODULE_ENABLE_VTK_ImagingStatistics=YES
+    -DVTK_MODULE_ENABLE_VTK_ImagingStencil=YES
+    -DVTK_MODULE_ENABLE_VTK_ImagingMath=YES
+    -DVTK_MODULE_ENABLE_VTK_InteractionImage=YES
+    -DVTK_MODULE_ENABLE_VTK_InteractionStyle=YES
+    -DVTK_MODULE_ENABLE_VTK_IOGeometry=YES
+    -DVTK_MODULE_ENABLE_VTK_IOImage=YES
+    -DVTK_MODULE_ENABLE_VTK_IOLegacy=YES
+    -DVTK_MODULE_ENABLE_VTK_IOMovie=YES
+    -DVTK_MODULE_ENABLE_VTK_IOXML=YES
+    -DVTK_MODULE_ENABLE_VTK_RenderingAnnotation=YES
+    -DVTK_MODULE_ENABLE_VTK_RenderingContext2D=YES
+    -DVTK_MODULE_ENABLE_VTK_RenderingContextOpenGL2=YES
+    -DVTK_MODULE_ENABLE_VTK_RenderingLOD=YES
+    -DVTK_MODULE_ENABLE_VTK_RenderingOpenGL2=YES
+    -DVTK_MODULE_ENABLE_VTK_ViewsContext2D=YES
+  VTK_DIR: "${{ github.workspace }}/vtk-build"
+  ITK_TAG: "v6.0a03"
+  ITK_MODULES: >-
+    -DModule_ITKVtkGlue=ON
+    -DModule_RTK=ON
+
+  ITK_DIR: "${{ github.workspace }}/itk-build"
 
 jobs:
-  build-ubuntu:
-    runs-on: ubuntu-latest
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-14]
```

Comment:
please change `ubuntu-latest` to `ubuntu-22.04` to prevent the glibc incompatibility.<br/>
`ubuntu-22.04` should still be available for ~2 years.